content/espanol/python-guess-number/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Python: Juego de Adivinar el Número"
+description: "Crea un juego sencillo que le pida al usuario adivinar un número generado aleatoriamente"
+date: 2020-07-27
+prereq: "Fundamentos de Python"
+difficulty: "Intermedio"
+icon: "fab fa-python"
+draft: false
+---
+
+En este taller, crearás un juego sencillo que incorpora algunos de los conceptos que aprendiste en la sección de Fundamentos de Python. ¡Prueba esta demostración y mira cuántos intentos te lleva adivinar el número!
```

Comment:
En este taller, crearás un juego sencillo que incorpora algunos de los conceptos que aprendiste en la sección de Fundamentos de Python. ¡Prueba esta demostración y descubre cuántos intentos te lleva adivinar el número!

Reply:
updated and resolved